controllers/session.go

```diff
@@ -66,3 +66,65 @@ func (c *ApiController) GetSessions() {
 		c.ResponseOk(sessions, paginator.Nums())
 	}
 }
+
+// AddUserSession
```

Comment:
What is the use of api addSession? We can not request the interface to add a session, only after the backend beego created, then casdoor to add a session

Reply:
It's for application to create a session at Casdoor